src/lib/common/sem.cpp

```diff
@@ -83,14 +85,39 @@ int semInit(int shared, int takenInitially)
 *
 * reqSemTake -
 */
-int reqSemTake(const char* who, const char* what)
+int reqSemTake(const char* who, const char* what, SemRequestType reqType, bool* taken)
 {
   int r;
 
+  if (reqType == SemNoneOp)
```

Comment:
This is just a "theoretical" case, right? I mean, we don't have any SemNoneOp operation...